src/main.ts

```diff
@@ -1,8 +1,18 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  const config = new DocumentBuilder()
+    .setTitle('Users')
+    .setDescription('The Users API description')
+    .setVersion('0.1')
```

Comment:
I would move swagger configuration to some config / service and in `boostrap()` just call a method to initialize it - so that `bootstrap` doesn't get to bloated when other services configurations arrive.